src/Plugin/Block/ContactsEntity.php

```diff
@@ -205,14 +205,23 @@ public function getEditLink($mode) {
   public function processManageMode(array &$variables) {
     $definition = $this->getPluginDefinition();
 
-    $variables['attributes']['data-contacts-manage-entity-type'] = $variables['entity'] = $definition['_entity_type_id'];
-    $variables['attributes']['data-contacts-manage-entity-bundle'] = $variables['bundle'] = $definition['_bundle_id'];
-
-    $variables['footer']['links'] = [
+    $variables['content']['links'] = [
       '#theme' => 'item_list',
       '#list_type' => 'ul',
       '#items' => $this->getManageLinks(),
     ];
+
+    if (!empty($definition['_required_hats'])) {
```

Comment:
Should this be on `ContactsEntity`, or should this be part of the manage mode code as it's not block specific?

Reply:
hmmm... No I think this is the right place for it. ContactsEntity is a dashboard block, and blocks can decide they require certain hats. So far its just profiles that do that, but I can image other scenarios.

Reply:
So although `ContactsEntity` is the only block that currently supports it, putting it here means that other blocks _can't_ support it. Having said that - not a priority for now, so let's put an `@todo` in with a link to an issue to decide/change this.